.github/workflows/browser-tests.yml

```diff
@@ -57,17 +62,24 @@ jobs:
 
             - name: Checkout PR branch
               uses: actions/checkout@v4
+              with:
+                  fetch-depth: 2
 
             - name: Setup Job and Install Dependencies
               uses: ./.github/actions/setup-job
 
             - uses: actions/download-artifact@v4
+              with:
+                  pattern: tachometer-results-*
+                  merge-multiple: true
 
             - name: Post Tachometer Performance Comment
-              uses: actions/github-script@v7
+              id: comment
+              run: echo "TACHOMETER_COMMENT=$(yarn log:performance | xargs)" >> "$GITHUB_OUTPUT"
+
+            - name: Post Performance to PR
+              uses: marocchino/sticky-pull-request-comment@v2
```

Comment:
We use this package over in CSS and it's very effective at posting and maintaining the comment without us having to maintain the code for doing so in our project.